app/models/user.js

```diff
@@ -195,7 +157,14 @@ exports.addModel = function(database) {
       that.id = uuid.v4()
 
       that.validateOnCreate()
-        .then(function(user) { return user.updateHashedPassword() })
+        .then(function(user) {
+          if (_.isFunction(user.hashedPassword)) {
+            var pwd = user.hashedPassword();
+            return user.updatePassword(pwd, pwd);
+          } else {
+            resolve(user);
```

Comment:
This is a personal choice to drop semicolons in javascript: https://github.com/pepyatka/pepyatka-server/wiki/Code-style